LiskHQ/lisk-sdk
# Reject unnecessary messages - Closes #4424

### What was the problem?

Peers can send unnecessary or internal messages that are not part of the protocol.

### How did I solve it?

Rejecting the message and banning the peer.

### How to manually test it?

Send messages like `INVALID` or something and see if the receiving node rejects the connection

### Review checklist

- [x] The PR resolves #4424
